objects/src/errors.rs

```diff
@@ -0,0 +1,34 @@
+use core::fmt;
+
+// ACCOUNT ERROR
+// ================================================================================================
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub enum AccountError {
+    AccountIdInvalidFieldElement(String),
+    AccountIdTooFewTrailingZeros,
+    FungibleFaucetIdInvalidFirstBit,
+}
+
+impl AccountError {
+    pub fn account_id_invalid_field_element(msg: String) -> Self {
+        Self::AccountIdInvalidFieldElement(msg)
+    }
+
+    pub fn account_id_too_few_trailing_zeros() -> Self {
+        Self::AccountIdTooFewTrailingZeros
+    }
+
+    pub fn fungible_faucet_invalid_first_bit() -> Self {
```

Comment:
nit: should this be `fungible_faucet_id_invalid_first_bit` to be consistent with the others and the error?

Reply:
Yes. Fixed!